### PR TITLE
Updates to storage doc

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Update about storage quota available on Safari
- Update about separate storage buckets on mobile Safari for installed PWA vs Safari
- Include Chrome DevTools feature for simulating limited storage

### CODE REVIEW

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 1]
```diff
@@ -159,7 +163,14 @@ of stored cross origin resources.
 During development, you can use your browser's DevTools to inspect the
 different storage types, and easily clear all stored data.
 
-<img class="w-screenshot w-screenshot-filled" alt="Storage test tool." src="storage-test-tool.png">
+A new feature was added in Chrome 88 that lets you override the storage quota
+site in teh Storage Pane. This feature gives you the ability to simulate
```
Comment (filipmares) [inline src/site/content/en/blog/storage-for-the-web/index.md:167]: Typo: "the Storage Pane"

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 2]
```diff
@@ -107,9 +107,13 @@ device.
   [may use up to 2GB][ff-usage-limits]. You can use the
   [StorageManager API](#check-available) to determine how much space is still
   available.
-* Safari (both desktop and mobile) appears to allow up to 1GB. When the limit
+* Safari (both desktop and mobile) appears to allow about 1GB. When the limit
   is reached, Safari will prompt the user, increasing the limit in 200MB
   increments. I was unable to find any official documentation on this.
+  * If a PWA is added to the home screen on mobile Safari, it appears to
+    create a new storage container, and nothing is shared between the PWA and
```
Comment (tomayac) [inline src/site/content/en/blog/storage-for-the-web/index.md:114]:
```suggestion
    create a new storage container, and nothing is shared between the PWA
```

Comment (tomayac) [reply src/site/content/en/blog/storage-for-the-web/index.md:114]: Double “and”

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 1 above)
Comment (tomayac) [inline src/site/content/en/blog/storage-for-the-web/index.md:167]:
```suggestion
site in the Storage Pane. This feature gives you the ability to simulate
```

Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md]: Fixed, thanks

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 3]
```diff
@@ -159,7 +163,14 @@ of stored cross origin resources.
 During development, you can use your browser's DevTools to inspect the
 different storage types, and easily clear all stored data.
 
-<img class="w-screenshot w-screenshot-filled" alt="Storage test tool." src="storage-test-tool.png">
+A new feature was added in Chrome 88 that lets you override the storage quota
+site in the Storage Pane. This feature gives you the ability to simulate
```
Comment (jpmedley) [inline src/site/content/en/blog/storage-for-the-web/index.md:167]: Override the storage quota site? Do you mean override the site's storage quota?

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 4]
```diff
@@ -159,7 +163,14 @@ of stored cross origin resources.
...
+different devices and test the behavior of your apps in low disk availability
+scenarios. Go to **Application** &gt; **Storage**, enable the
```
Comment (jpmedley) [inline src/site/content/en/blog/storage-for-the-web/index.md:169]:
```suggestion
scenarios. Go to **Application** then **Storage**, enable the
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 3 above)
Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md]: Yep, fixed

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 4 above)
Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md]: fixed

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 5]
```diff
@@ -159,7 +163,14 @@ of stored cross origin resources.
...
 
-<img class="w-screenshot w-screenshot-filled" alt="Storage test tool." src="storage-test-tool.png">
+A new feature was added in Chrome 88 that lets you override the site's storage
+quota in the Storage Pane. This feature gives you the ability to simulate
+different devices and test the behavior of your apps in low disk availability
+scenarios. Go to **Application** then **Storage**, enable the
+**Simulate custom storage quota** checkbox, and enter any valid number to
+simulate the storage quota.
+
+<img class="w-screenshot w-screenshot-filled" alt="DevTools Storage pane." src="devtools-storage-panel.png">
```
Comment (samthor) [inline src/site/content/en/blog/storage-for-the-web/index.md:173]: This change seems fine but could you upload the image via the new [flow](https://web.dev/handbook/markup-media/)?

**Review by samthor — APPROVED** [review]

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 5 above)
Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md]: Done.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**